packages/app/src/systems/DApp/components/index.tsx

```diff
@@ -1 +1,2 @@
 export * from './ConnectInfo';
+export * from '../../Transaction/components/TxContent';
```

Comment:
Maybe we need to fix the imports here, as it looks bad to export on index a component from other place.